client/src/lib/components/buy/buy-hud.svelte

```diff
@@ -28,9 +28,9 @@
     uiStore.showModal = true;
     uiStore.modalData = {
       location: hexStringToNumber($selectedLandMeta!.location),
-      sellPrice: $selectedLandMeta!.sellPrice!,
-      tokenUsed: $selectedLandMeta!.tokenUsed!,
-      tokenAddress: $selectedLandMeta!.tokenAddress!,
+      sellPrice: $selectedLandMeta!.sellPrice ?? 0,
```

Comment:
```suggestion
      // TODO: Enforce null checks here
      sellPrice: $selectedLandMeta!.sellPrice ?? 0,
```

In general, this goes against the [Fail Fast](https://www.martinfowler.com/ieeeSoftware/failFast.pdf) ideology, and this could bite us back sooner than later, but let's get this thing tested now.